robotis-op2/simulated-devices.md

```diff
@@ -37,27 +32,56 @@ robot, they are called `BackLedGreen`, `BackLedBlue` and `BackLedRed`.
 
 %end
 
-The corresponding position of each servo can be seen in figure
-[Actuator](#position-of-the-servos). Each of the 20 servos has the following
-configuration:
+Each of the 20 servos has the following configuration.
 
-%figure "Configuration of the servo of the DARwIn-OP"
-
-|                |      |        | | | | |  |
-| --------------- | ----- | ------- |  |  |  |  |  |
+%figure "Standard servo configuration"
+|                 |       |         |
+| --------------- | ----- | ------- |
 | maxForce        | 2.5   | N*m     |
 | acceleration    | 55    | rad/s^2 |
 | maxVelocity     | 12.26 | rad/s   |
 | dampingConstant | 0.002 |         |
 | staticFriction  | 0.025 | N*m     |
-
 %end
 
-%figure "Position of the servos"
+### LEDs
 
-![DARwIn-OP_Actuator.png](images/DARwIn-OP_Actuator.png)
+The two RGB LEDs are placed on the robot head. They are called `HeadLed` and
+`EyeLed`. The two eyes will have same color. Note that these LEDs are
+replicated on the robot back panel.
 
-%end
+The three other LEDs are unicolor and placed on the robot back.
+They are called `BackLedGreen`, `BackLedBlue` and `BackLedRed`.
+
+### Accelerometer
+
+The accelerometer returns values between 0 and 1024 corresponding to values
+between -3 [g] to +3 [g] like on the real robot.
+Its name is `Accelerometer`.
```

Comment:
To me this sentence is not very clear. I would rather write something like:
``The Webots device simulating it is called 'Accelerometer'``

Same for the gyroscope and speaker